### PR TITLE
feat: dynamic config yaml

### PR DESCRIPTION
**Pull request by sergeyWh1te**
## Summary

Dynamic config

### CODE REVIEW

File: notification.protocol.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    webhook_url: "YOUR_DISCORD_WEBHOOK_URL_2"
+
+opsgenie_channels:
+  - id: "OpsGenie1"
+    description: "OpsGenie channel for high and critical messages"
+    api_key: "YOUR_OPSGENIE_API_KEY_1"
+
+teams:
+  - name: "protocol"
+    nats_subject: "nats.protocol.findings"
```
Comment (arwer13) [inline notification.protocol.yaml:43]: it's different now `findings.nats.${ctx.config.team}`

File: notification.protocol.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    description: "Telegram channel for all messages >= low, no quorum"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_2"
+    chat_id: "YOUR_CHAT_ID_2"
+  - id: "Telegram3"
+    description: "Telegram channel for messages >= low, with quorum"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_3"
+    chat_id: "YOUR_CHAT_ID_3"
+  - id: "Telegram4"
+    description: "Telegram channel for high and critical messages, with quorum"
+    token: "YOUR_TELEGRAM_BOT_TOKEN_4"
```
Comment (arwer13) [inline notification.protocol.yaml:25]: I'd name `token` as `botToken` for clarity

File: notification.protocol.yaml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    description: "Discord channel for all messages >= low, no quorum"
+    webhook_url: "YOUR_DISCORD_WEBHOOK_URL_2"
+
+opsgenie_channels:
+  - id: "OpsGenie1"
+    description: "OpsGenie channel for high and critical messages"
+    api_key: "YOUR_OPSGENIE_API_KEY_1"
+
+teams:
+  - name: "protocol"
```
Comment (arwer13) [inline notification.protocol.yaml:42]: Good!: with the dynamic config we can get rid of staging code and replace it with `protocolState` team config entry

File: notification.protocol.yaml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        type: "Telegram"
+        channel_id: "Telegram1"
+        severity_refs:
+          - "unknown"
+        by_quorum: false
+
+      - consumerName: "TelegramDebug"
+        type: "Telegram"
+        channel_id: "Telegram2"
+        severity_refs:
```
Comment (arwer13) [inline notification.protocol.yaml:56]: Short notation would be ok to yse: `severity_refs: [ low, medium, high, critical ]`

File: notification.protocol.yaml (hunk 1 above)
Comment (sergeyWh1te) [reply notification.protocol.yaml]: Yes, I 've changed topics mask

File: notification.protocol.yaml (hunk 2 above)
Comment (sergeyWh1te) [reply notification.protocol.yaml]: Renamed

File: notification.protocol.yaml (hunk 3 above)
Comment (sergeyWh1te) [reply notification.protocol.yaml]: Removed teams.

File: notification.protocol.yaml (hunk 4 above)
Comment (sergeyWh1te) [reply notification.protocol.yaml]: It's work with that syntax too

**Pull request merged**